skultrafast/messpy.py

```diff
@@ -51,16 +53,18 @@ def process_vis(self,
             n_ir_cwl = data_file['wl_Remote IR 32x2'].shape[0]
             para_idx = np.repeat(np.array([False, True], dtype='bool'),
                                  n_ir_cwl)
-
+        if invert_data:
+            d *= -1
         dpm = sigma_clip(d[para_idx, ...], axis=0, sigma=sigma)
         dsm = sigma_clip(d[~para_idx, ...], axis=0, sigma=sigma)
         dp = dpm.mean(0)
-        dps = dpm.std(0)
         ds = dsm.mean(0)
-        dss = dsm.std(0)
-
-        ds -= ds[:, :10, ...].mean(1, keepdims=True)
-        dp -= dp[:, :10, ...].mean(1, keepdims=True)
+        #dps and dss are not yet used
+        #dps = dpm.std(0)
+        #dss = dsm.std(0)
+        if BG_points:
```

Comment:
After short thinking, the number of the BG points should be in the datafile itself. Therefore this should not be necessary for messpy2d. Additionally, one can always use the subtract background method of TimeResSpec if necessary.

Reply:
Which would make sense. However right now BG is subtracted by default using the first 10 points - for now I would either take it out and do it with subtract background or have it as input.